src/constants/index.js

```diff
@@ -795,3 +795,10 @@ export const clients = [
     logo: dropbox,
   },
 ];
+
+export const aboutMe = [
+  {
+    name: "Parth",
+    lastName: "Mittal"
+  }
+];
```

Comment:
```suggestion
export const aboutMe = {
    name: "Parth Mittal"
  }
```

I feel an object is more extensible as indices won't be required.

Reply:
done!